pom.xml

```diff
@@ -84,10 +84,11 @@
         <module>enrollment-server-onboarding-domain-model</module>
         <module>enrollment-server-onboarding-provider-iproov</module>
         <module>enrollment-server-onboarding-provider-zenid</module>
+        <module>enrollment-server-onboarding-provider-innovatrics</module>
     </modules>
 
     <properties>
-        <openapi-generator-maven-plugin.version>6.5.0</openapi-generator-maven-plugin.version>
+        <openapi-generator-maven-plugin.version>7.0.0</openapi-generator-maven-plugin.version>
```

Comment:
Ok, why not. I usually prefer to update dependencies in a separate PR but as far as I remember, you mentioned, that the update is required because of some specific grammar, right?

Reply:
Yes. Version 7.0.0 was the first one after 6.5.0 that handled the innovatrics classes generation.